dataactcore/aws/s3UrlHandler.py

```diff
@@ -119,3 +118,23 @@ def getFileSize(filename):
             return False
         else:
             return key.size
+
+    def getFileUrls(self, bucket_name, path):
+        try:
+            s3UrlHandler.REGION
+        except AttributeError as e:
+            s3UrlHandler.REGION = CONFIG_BROKER["aws_region"]
+
+        path = "".join([path, "/"]) if path[-1] != "/" else path
```

Comment:
Does it make sense to do this type of checking and correcting in `dataactcore/config.py` so that all downstream references can assume that `help_files_path` is correct?